src/lightning_lite/accelerators/__init__.py

```diff
@@ -13,6 +13,7 @@
 from lightning_lite.accelerators.accelerator import Accelerator  # noqa: F401
 from lightning_lite.accelerators.cpu import CPUAccelerator  # noqa: F401
 from lightning_lite.accelerators.cuda import CUDAAccelerator  # noqa: F401
+from lightning_lite.accelerators.cuda import find_usable_cuda_gpus  # noqa: F401
```

Comment:
IMO this utility should be left at the `cuda` level

Reply:
The import would be too long:
`lightning.fabric.accelerators.cuda`